src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -90,6 +103,28 @@ public override void OnStartActivity(Activity activity, object payload)
                 span.PutHttpUserAgentAttribute(userAgent);
                 span.PutHttpRawUrlAttribute(GetUri(request));
             }
+
+            if (newActivity != null)
+            {
+                if (newActivity.IsAllDataRequested)
+                {
+                    if (request.Host.Port == 80 || request.Host.Port == 443)
+                    {
+                        newActivity.AddTag("http.host", request.Host.Host);
```

Comment:
Why do you not include the port for 80/443?

Reply:
just retaining the current behavior. Current behavior is _probably_ because 80,443 are common and can be deducted from other info like url etc.